BranchMetrics/cpp-branch-deep-linking-attribution
# SDK-599 -- Add ability to launch Windows application from web link

## Reference
SDK-599 -- Add ability to launch Windows application from web link

## Description
We need to be able to click on a web link, and have a Windows sample application open.

After this change, the ColorPicker sample responds to a URI that looks like color://foo/bar

*HOWEVER*
This works by creating registry keys under HKEY_CURRENT_USER\Software\Classes\color.  To set up the registry keys, I added a "File -> Install" menu option that does the right thing.

## Testing Instructions
1. Build and run the latest Windows ColorPicker Sample
2. Dismiss the color dialog that opened the first time
3. Select File -> Install
4. Exit the app.
5. (optional) Examine the Registry to see the pieces that were required to make this work
6. In Chrome, type in a url that looks like color://foo/bar

Expected:  The Color Picker will open (after it prompts you the first time if that is what you want)

## Risk Assessment [`MEDIUM`]
This is fairly self-contained.  It will either work or not. 


## Reviewer Checklist (To be checked off by the reviewer only)

- Correctness & Style
